apps/skolplattformen-sthlm/android/app/src/debug/java/com/app/CookieInterceptor.java

```diff
@@ -0,0 +1,82 @@
+package com.app;
+
+import android.util.Log;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import kotlin.Pair;
+import okhttp3.Cookie;
+import okhttp3.CookieJar;
+import okhttp3.HttpUrl;
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+class CookieInterceptor implements Interceptor {
+  private final List<Cookie> cookies;
+  private final CookieJar cookieJar;
+
+  public CookieInterceptor(CookieJar cookieJar) {
+    this.cookies = new ArrayList<>();
+    this.cookieJar = cookieJar;
+  }
+
+  @NotNull
+  @Override
+  public Response intercept(@NotNull Chain chain) throws IOException {
+    String domain = chain.request().url().topPrivateDomain();
+    Log.d("Skolplattformen", "requseting resource on domain: " + domain);
+    if(domain == null || !domain.contains("goteborg.se") && !domain.contains("funktionstjanster.se")) {
+      return chain.proceed(chain.request());
+    }
+
+    Log.d("Skolplattformen", "\n\n<<<<<<<<<<<<<<<<<<<<< BEGIN >>>>>>>>>>>>>>>>>>");
```

Comment:
Most of this is just to simplify debugging of the request/response headers by running `adb logcat -s Skolplattformen` to